bentoml/_internal/log.py

```diff
@@ -88,7 +88,7 @@ def configure_logging():
         CLI_LOGGING_CONFIG["root"]["level"] = logging.DEBUG
     else:
         CLI_LOGGING_CONFIG["loggers"]["bentoml"]["level"] = logging.INFO
-        CLI_LOGGING_CONFIG["root"]["level"] = logging.WARNING
+        CLI_LOGGING_CONFIG["root"]["level"] = logging.INFO
```

Comment:
This should be INFO so that it doesn't quiet server script logs

Reply:
Why aren't the server script logs logging from the bentoml namespace?

Reply:
We should configure those logs manually; we don't want to allow info logs from any other libraries through by default, I think.